nodeup/pkg/model/sysctls.go

```diff
@@ -195,5 +197,38 @@ func (b *SysctlBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 		OnChangeExecute: [][]string{{"sysctl", "--system"}},
 	})
 
+	// Running AWS VPC CNI on Ubuntu 22.04 Requires Setting MACAddressPolicy to `none` (ref: https://github.com/aws/amazon-vpc-cni-k8s/issues/2103 & https://github.com/kubernetes/kops/issues/16255)
+	klog.V(2).Infof("Checking if we need to disable MACAddressPolicy on Ubuntu 22.04")
```

Comment:
Please move this block to `nodeup/pkg/model/networking`. This is where CNI-specific code is located.